math/src/main/java/com/powsybl/math/matrix/SparseMatrix.java

```diff
@@ -252,6 +295,29 @@ public void add(int i, int j, double value) {
         }
     }
 
+    /**
+     * {@inheritDoc}
+     *
+     * <p>
+     * As sparse matrix is stored in CSC format. Columns must be filled in ascending order but values inside a column
+     * may be filled in any order.
+     * </p>
+     * @throws PowsyblException if values are filled in wrong order.
+     */
+    @Override
+    public Element addAndGetElement(int i, int j, double value) {
+        add(i, j, value);
+        return new SparseElement(values.size() - 1);
+    }
+
+    /**
+     * {@inheritDoc}
```

Comment:
But maybe we should mention that it does not clear the elements, only set their values to zero ?

Reply:
We can also rename the method something like fillWithZeros ?

Reply:
Ok it may be more explicit. But still the "natural" way to do this with a sparse matrix would be to delete the elements :)
I think there is no real good naming here : anyway the difference between setting an element value to zero and deleting it only makes sense for a sparse matrix, so it cannot be clear in a generic Matrix API.

Reply:
Ok so we can keep it as it is and we will see later how to refactor the overall design of Matrix API